<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPNManager</class>
 <widget class="QMainWindow" name="VPNManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>507</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VPNManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btServers">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Servers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btkeys">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Keys</string>
    </property>
   </widget>
   <widget class="QListView" name="listServers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>.ovpn files</string>
    </property>
   </widget>
   <widget class="QListView" name="listKeys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>.keys files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textServers">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>10</y>
      <width>411</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Folder where .ovpn files are</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>OVPN Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textKeys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>411</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Folder where .key files are</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Key Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btConnect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>password</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QListView" name="listCA">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>.crt files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textCA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>411</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Folder where .crt files are</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>CA Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btCA">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textStatus">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Folder where .crt files are</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Not Connected!</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDisconnect">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPID">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Folder where .crt files are</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPID">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btClose">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
